src/openrct2/world/scenery.h

```diff
@@ -32,6 +32,8 @@
 #define SCENERY_PATH_SCENERY_ID_MIN  0x100
 #define SCENERY_PATH_SCENERY_ID_MAX  0x10F
 
+#define SCENERY_WINDOW_TABS 20 // (MAX_SCENERY_GROUP_OBJECTS + 1). The + 1 is for the 'Miscellaneous' tab
```

Comment:
I can live with this for now, but is there any chance on getting this back to `(MAX_SCENERY_GROUP_OBJECTS + 1)` at some point?

Reply:
I will have a look at this after the PR is merged.

Reply:
Could a default tab that's empty be altered in a future pr? Like if one deselected every type of vegetation they serve no purpose.

It's weird having an empty tab